amqp/src/main/scala/akka/stream/alpakka/amqp/AmqpRpcFlowStage.scala

```diff
@@ -0,0 +1,179 @@
+package akka.stream.alpakka.amqp
+
+import akka.stream._
+import akka.stream.stage._
+import akka.util.ByteString
+import com.rabbitmq.client.AMQP.BasicProperties
+import com.rabbitmq.client.{DefaultConsumer, Envelope, ShutdownSignalException}
+
+import scala.collection.mutable
+import scala.concurrent.{Future, Promise}
+
+object AmqpRpcFlowStage {
+
+  /**
+    * Internal API
```

Comment:
No need to mark it Internal when it is `private` as it will not be accessible anyways. We use it when something is `private[alpakka]` for example, since that does not have a Java access control counterpart and will be accessible from Java.